doc/nrf/test_and_optimize.rst

```diff
@@ -153,6 +154,8 @@ Use one of the following methods:
 The |NCS| provides a common set of features and options for debugging and optimizing the application performance.
 This includes a multilevel logging system that can be enabled and configured independently for different modules and logging backends (such as :ref:`UART, J-Link RTT or Spinel <ug_logging_backends>`), and support for writing unit tests using Unity and CMock.
 
+If you want to go through dedicated training related to some of the topics covered here, enroll in the `nRF Connect SDK Intermediate course`_ in the `Nordic Developer Academy`_.
```

Comment:
This should be a note, like the similar line on https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation.html .

Also, can we have this update as a separate commit in this PR? It is not really related to the release notes.